l10n_de_website_sale_accept_terms/views/views.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <!-- Add needed CSS and JS in template -->
+        <template id="assets_frontend" inherit_id="website.assets_frontend" name="Accept Terms on Checkout">
+          <xpath expr="." position="inside">
+                <link rel='stylesheet' href='/l10n_de_website_sale_accept_terms/static/src/css/style.css'/>
+                <script type="text/javascript" src="/l10n_de_website_sale_accept_terms/static/src/js/script.js"></script>
+          </xpath>
+        </template>
+
+        <!-- Replace the payment-site -->
```

Comment:
Yes, it is worth a further discussion.

This is what I changed:

Normally the whole acquirer-button-code (selectable payment methodes) is after the total amount. But this is not allowed in Germany.
To make it in consistency with our laws I seperated the code in two pieces:

https://github.com/copado/l10n-germany/blob/8.0/l10n_de_website_sale_accept_terms/views/views.xml#L34-L48

https://github.com/copado/l10n-germany/blob/8.0/l10n_de_website_sale_accept_terms/views/views.xml#L104-L111

And also a highlight around the products and prices:
https://github.com/copado/l10n-germany/blob/8.0/l10n_de_website_sale_accept_terms/views/views.xml#L57

With my previous knowledge about Odoo I don't find a way to do this in an extension.
But I can give a try a chance to do this with some xpath's.